xtext-core.target

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?>
+<target name="xtext-core" sequenceNumber="0">
+	<locations>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="https://download.eclipse.org/modeling/tmf/xtext/updates/milestones/"/>
+			<unit id="org.eclipse.xtend.sdk.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.xtend" version="0.0.0"/>
+			<unit id="org.eclipse.xtend.typesystem.emf" version="0.0.0"/>
+		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="https://download.eclipse.org/modeling/emf/emf/builds/release/2.29"/>
+			<unit id="org.eclipse.emf.sdk.feature.group" version="0.0.0"/>
+		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="https://download.eclipse.org/modeling/emft/mwe/updates/releases/2.14.0/"/>
+			<unit id="org.eclipse.emf.mwe2.language.sdk.feature.group" version="0.0.0"/>
+		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="https://download.eclipse.org/releases/2022-12/"/>
```

Comment:
Yes, it's on the TODO list I was about to document.
I had to switch to 2022-12 because with 2022-03 I couldn't run plug-in tests from Eclipse (Eclipse wasn't starting at all, though the CPU was doing something; I haven't investigated further). Actually, there's no need to run plug-in tests in this Git repository, but just in case.